k8s-bench/tasks/find-application-level-failures/artifacts/eval-app.py

```diff
@@ -0,0 +1,20 @@
+import sys
+import time
+
+print("Starting eval-app...")
+sys.stdout.flush()
+
+counter = 0
+while True:
+    counter += 1
+    if counter % 4 == 0:
```

Comment:
every 4 seconds makes it very obv. would be good to use some sort of exponential back-off to make it less frequent.